src/clr-addons/quick-list/quick-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div [ngClass]="onCard?'card':''">
+  <div class="clr-row">
+    <div *ngIf="!!label" class="clr-col-4">
+      <div class="clr-row">
+        <label class="pt-1 pl-1">{{label}}</label>
+      </div>
+    </div>
+    <div [ngClass]="!!label?'clr-col-8':'clr-col-12 pl-1'">
```

Comment:
handle the width classes like in combobox (see controlClasses Input and method) to be more flexible and clarity standard conform